IPython/core/page.py

```diff
@@ -210,7 +210,7 @@ def pager_page(strng, start=0, screen_lines=0, pager_cmd=None):
                 retval = None
                 # if I use popen4, things hang. No idea why.
                 #pager,shell_out = os.popen4(pager_cmd)
-                pager = os.popen(pager_cmd, 'w')
+                pager = os.popen(pager_cmd + ' 2>/dev/null', 'w')
```

Comment:
Hum, wondering if we should use `subprocess.Popen` . Potentially `subprocess.run` now that this is available and properly discard stderr. I'm worried about string munging... but I may just be lazy and merge that as is later.

Reply:
I don't have enough experience with passing stdin to `subprocess.Popen` to be sure to cover all the edge-cases are covered in the same way they are covered now.

Reply:
> I don't have enough experience with passing stdin to `subprocess.Popen` to be sure to cover all the edge-cases are covered in the same way they are covered now.

Then don't worry I'll push a commit on this branch if I decide `subprocess.Popen` is better.

Maybe not this week though.

Reply:
Friendly ping :)

Reply:
Yes sorry Aug/Sep/Oct were crazy with my team size at work decreasing by 33%... Slowly getting out of the tarpit.